tests/rag/test_rag.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from typing import Generator, Any
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.namespace import Namespace
+from ocp_resources.deployment import Deployment
+
+
+@pytest.fixture(scope="function")
```

Comment:
Could this fixture be also in tests/rag/conftest.py for consistency?

Reply:
Done. :+1: